.eslintrc.js

```diff
@@ -1,40 +1,69 @@
+/** @format */
+
 module.exports = {
 	root: true,
-	'extends': [
-		'wpcalypso/react',
-		'plugin:jsx-a11y/recommended',
-		'plugin:jest/recommended',
-	],
+	extends: [ 'wpcalypso/react', 'plugin:jsx-a11y/recommended', 'plugin:jest/recommended' ],
 	parser: 'babel-eslint',
 	env: {
 		browser: true,
 		'jest/globals': true,
+		// mocha is only still on because we have not finished porting all of our tests to jest's syntax
 		mocha: true,
-		node: true
+		node: true,
 	},
 	globals: {
+		// this is our custom function that's transformed by babel into either a dynamic import or a normal require
 		asyncRequire: true,
+		// this is the name of the project from the build config. Injected at boot in a script tag.
 		PROJECT_NAME: true,
+		// this is the SHA of the current commit. Injected at boot in a script tag.
 		COMMIT_SHA: true,
 	},
-	plugins: [
-		'jest',
-		'jsx-a11y',
-		'import',
-	],
+	plugins: [ 'jest', 'jsx-a11y', 'import' ],
 	rules: {
-		camelcase: 0, // REST API objects include underscores
+		// REST API objects include underscores
+		camelcase: 0,
+
+		// TODO: why did we turn this off?
 		'jest/valid-expect': 0,
-		'jsx-a11y/anchor-has-content': 0, // i18n-calypso translate triggers false failures
+
+		// i18n-calypso translate triggers false failures
+		'jsx-a11y/anchor-has-content': 0,
+
+		// error if any given line is over 140 characters long.
+		// Ideally this would be even lower(120, 80?), but that's a big step.
 		'max-len': [ 2, { code: 140 } ],
```

Comment:
Shall we disable this as well? Prettier's functionality is based on breaking long lines.

[A quick search](https://github.com/Automattic/wp-calypso/search?utf8=%E2%9C%93&q=max-len&type=) will show lots of disabling for SVG paths or strings that aren't broken.

Reply:
sounds good to me. I'll rename this PR.